.travis.yml

```diff
@@ -4,6 +4,12 @@ node_js:
 addons:
   chrome: stable
   firefox: latest
+notifications:
+  slack:
+    secure: JKzk2sJSbZ9h2PUVWj6KtOAdFbEEnOtv/VZy05pJ2H41xRgUHiGdtMW/vMSeq6XX3IJN8eW2zd0cJTgkFn0ioAlYvID8zRhcvkFHg60QXquoqtp5y65dxjtVz79hefxSo7FO1NhMZBQWE9Tg6R7XkoyTMth62+T9vqOgu2Hms6M=
+    if: branch = main
+    on_success: never # default: change
```

Comment:
> ... notify on success when the value changes back.

With `on_success` set to `never`, will this happen?

Reply:
Apologies, non-email integration appear to have a different default from emails and I was testing something out. This has been updated to be inline with the PR description.